deployed-contracts/oracles.md

```diff
@@ -101,3 +101,19 @@
 * DefaultProxyAdmin:[`0xA78A1Df376c3CEeBC5Fab574fe6EdDbbF76fd03e`](https://sepolia.arbiscan.io/address/0xA78A1Df376c3CEeBC5Fab574fe6EdDbbF76fd03e)
 * OneJump Oracle weETH/eETH/USD (having intermediate oracle Chainlink): [`0x4f894aC6F5e2E2bD72632B137E28234339Db3D53`](https://sepolia.arbiscan.io/address/0x4f894aC6F5e2E2bD72632B137E28234339Db3D53)
 * OneJump Oracle wstETH/stETH/USD (having intermediate oracle Chainlink): [`0x06b47B32d9C4D8f948EB3FFDB4a1ecE93794f922`](https://sepolia.arbiscan.io/address/0x06b47B32d9C4D8f948EB3FFDB4a1ecE93794f922)
+
+## Zksync Sepolia
```

Comment:
Switch the order: first zkSync mainnet, then zkSync sepolia

Reply:
[a641fa1](https://github.com/VenusProtocol/venus-protocol-documentation/pull/224/commits/a641fa1f85c152de1f79e104492ec5fecbf16b04)